packages/nextjs/app/builders/page.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import MembersList from "./_components/MembersList";
+import fs from "fs/promises";
+import path from "path";
+
+// Getting teh buildres dir. data
+const getBuildersData = async () => {
```

Comment:
I still think it would be cool to sort this list data to group everyone who has a personal page first in the array. That way the table you pass it to would highlight everyone with pages, making it easier to learn more about them.

Reply:
Sorted the Builders array so that those with personal pages now appear at the top.

<img width="1004" alt="Screenshot 2024-08-22 at 2 30 48 PM" src="https://github.com/user-attachments/assets/42868efa-e312-479c-9ac0-1ee68d1c24f9">